triggers-utils/create-ingress.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-ingress
```

Comment:
It would be great if you could add the necessary metadata (see below) so that Tekton Hub and other supporting tools (in development) is able to search the task.

```yaml

metadata:
  labels:
    app.kubernetes.io/version: "0.1"
  annotations:
    tekton.dev/pipelines.minVersion: "0.12.1"   
    tekton.dev/tags: ingress, triggers
    tekton.dev/displayName: An Optional name for the task 

spec:
   description: >-
     One line summary of the task. 

      Description of the task 

```

see: #389